script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -36,11 +36,11 @@ the sight of my sister being scolded by that man,
 and even the sight of the doctor frolicking like a child,
 all of which, I remembered as though it were a dream.
 <Page11>
-"Huh? You're not actually awake, are you!?"
+"Eh? You're not actually awake, are you...?!"
```

Comment:
Eh is the direct japanese onomatopoeia but I think huh is better as a translation.

Added ellipses is ok.